fake_switches/cmd/main.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from twisted.internet import reactor
+from fake_switches.switch_configuration import SwitchConfiguration, Port
+from fake_switches.ssh_service import SwitchSshService
+from fake_switches.cisco.cisco_core import CiscoSwitchCore
+
+
+MODEL_MAPPING = {
+    'cisco_generic': CiscoSwitchCore
```

Comment:
I wanted to have "complete" switches with the appropriate port mappings, such as "Cisco_2960G-24TC-L", etc. etc.